lib/schema.js

```diff
@@ -527,11 +536,41 @@ class Metaschema {
   }
 }
 
+const validationSchemas = require('./validation-schemas');
+const schemaValidator = new Metaschema();
```

Comment:
We cannot do this, because in this case we need require `./schema.js` from `./validation-schemas.js` while `Metaschema` haven't been exported from `./schema.js` yet

Reply:
```js
let schemaValidator = null;
```
and then at the end of the file:
```js
schemaValidator = require('./validation-schemas');
```